### PR TITLE
📝(content) extension for edu

### PR DESCRIPTION
**Pull request by joycetoh8**
Adding extension development page to EDU content

### CODE REVIEW

File: site/en/education/extensions.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Extension development
+metadesc: An overview of Chrome extensions development for education on Chrome OS
+date: 2021-03-21
```
Comment (Snugug) [inline site/en/education/extensions.md:4]:
```suggestion
date: 2021-03-29
```

File: site/en/education/extensions.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+[Chrome extensions](https://developer.chrome.com/docs/extensions/mv3/) are software programs built on web technologies that customize the browser experience for a user.
+
+Extensions are run in a sandboxed environment but can interact with web content, which makes them suitable for some education use cases like:
+
+- browser filtering and monitoring
+- inserting specialized characters such as mathematical symbols and equations
+- content enrichment, for example for identifying spelling and grammatical errors
+- improve productivity (e.g. limiting time on social media)
+
+When building solutions for education use cases on Chromebooks, you can create an extension, a web app, or both. If you only plan to serve a single and specific purpose, then an extension is a good option to consider. Your extension can interact with web apps that you don’t own. For example, an extension may detect grammatical errors on a Google doc and provide comments and recommendations to fix them. Another extension may help students with note-taking by accumulating text the student highlights on any web page into one doc.
```
Comment (Snugug) [inline site/en/education/extensions.md:19]: The "single and specific purpose" reads a little weird to me, and should probably be hedged a little more, as most websites have a single purpose. Maybe hedge it in things that extensions can do that web apps can't. In general I think the advice should be to build an extension only if you need to run in that cross-site context

**Review by Snugug — CHANGES_REQUESTED** [review]
Thanks for your work here Joyce! I've added a few comments to help with repeated words and grammar.

File: site/en/education/extensions.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Developing Chrome OS extensions
+
+### Samples
+
+The Chrome Developers website has an excellent [getting started guide](https://developer.chrome.com/docs/extensions/mv3/getstarted/) to begin Chrome extension development. You should also check out samples available on the official Chrome [GitHub repository](https://github.com/GoogleChrome/chrome-extensions-samples).
+
+### APIs
+
+When developing extensions, there are extensive [APIs](https://developer.chrome.com/docs/extensions/reference/) to help you achieve the functionality you’re building.
```
[the remark this answers is not in the document]
Comment (joycetoh8) [reply site/en/education/extensions.md:35]: good catch!

File: site/en/education/extensions.md (hunk 2 above)
Comment (joycetoh8) [reply site/en/education/extensions.md:19]: Yeah, I agree it does read a little weird and vague. I was actually trying to reference the "single purpose" policy (see line 47) which requires extensions to have a narrow purpose. But I think your point about extensions running in a cross-site context is a better differentiator for extensions from web apps. Will make that edit, thanks!

**Review by nohe427 — APPROVED** [review]
LGTM

**Pull request merged**